src/internal/scheduler/animationFrame.ts

```diff
@@ -36,5 +46,99 @@ import { AnimationFrameScheduler } from './AnimationFrameScheduler';
  * @name animationFrame
  * @owner Scheduler
  */
+export const animationFrame: SchedulerLike = {
+  schedule<S>(work: (state: S, reschedule: (nextState: S) => void) => void, delay = 0, state?: S): Subscription {
```

Comment:
The previous implementation of `AnimationFrameScheduler` extended `AsyncScheduler` and got it's `schedule` implementation from the base class, so it supported `__rx_scheduler_overrides__`. The current implementation disregards `__rx_scheduler_overrides__` so it will behave differently under test - i.e. it'll fail, as it'll be async.